packages/shared/src/modules/users/libs/types/user-auth-response-dto.type.ts

```diff
@@ -1,8 +1,8 @@
+import { type UserInfoResponseDto } from "./user-info-response-dto.type.js";
+
 type UserAuthResponseDto = {
 	createdAt: string;
-	email: string;
 	id: number;
-	name: string;
-};
+} & UserInfoResponseDto;
```

Comment:
We need to either pick all these properties from UserGetAllItemResponseDto, including createdAt and id or leave it as it was, without combining types

Reply:
Okay, I get it. I'd rather leave it the way it was so as not to make a mess